internal/binder/function/funcs_stateful_test.go

```diff
@@ -49,13 +49,13 @@ func TestCompressExec(t *testing.T) {
 				"hello world",
 				"zlib",
 			},
-			result: []byte{120, 156, 202, 72, 205, 201, 201, 87, 40, 207, 47, 202, 73, 1, 4, 0, 0, 255, 255, 26, 11, 4, 93},
+			result: []byte{120, 156, 0, 11, 0, 244, 255, 104, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100, 3, 0, 26, 11, 4, 93},
```

Comment:
Is this decompressible by other lib or even languages like nodejs?

Reply:
yes , i tested it 
![image](https://user-images.githubusercontent.com/22807972/230576570-d757585f-e150-4f6e-96e2-31ce966955c9.png)